Dockerfile

```diff
@@ -23,4 +23,5 @@ ENV MEGAQC_PRODUCTION=1 \
 COPY . /app
 # Copy the compiled JS in from the other node container
 COPY --from=0 /app/megaqc/static/ /app/megaqc/static/
-RUN pip install /app[prod]
+
+RUN pip install /app[prod]
```

Comment:
Newline after the line, I mean, not before it. Note how the diff appears: 
![image](https://user-images.githubusercontent.com/5019367/99982443-aa7a0a80-2dfe-11eb-9eba-0cc953ef43e9.png)

Reply:
Ahh, whoopsie.